packages/vats/src/lib-board.js

```diff
@@ -9,18 +8,18 @@ import { crc6 } from './crc.js';
 import './types.js';
 
 const CRC_NUM_DIGITS = 2;
```

Comment:
out of scope but curious, why 2 digits?

Reply:
More than 1 but not too many? :p

Not for any good reason.  I think I did it earlier to try to help disambiguate in the face of typos.